test/test_integration.py

```diff
@@ -200,12 +200,25 @@
             'expect_out': 'b,c\n2,3\n5,6\n'}
 
 extra_newline = {'name': 'extra_newline',
-                 'raw_data': 'col1,col2,col3\n1,2\n,3\n',
+                 'raw_data': """Ref_ID	Full Reference
+Ref_34	"Emmo The
+University pp"
+Ref_36	"Fidi and
+Nancy J.d
+Eulemur st,
+southeast"
+Ref_37	"Emmons H.
+A new genus.
+American Museum 1999"
+""",
```

Comment:
I'm guessing this was an actual failing line, but for the integration tests it would seem more useful to distill this to a simpler more generic case like:

```
"""col1,col2,col3
"ab", "cd", "e\nf"
"""